pkg/aws/eni/node.go

```diff
@@ -494,9 +495,9 @@ func (n *Node) ResyncInterfacesAndIPs(ctx context.Context, scopedLog *logrus.Ent
 
 // GetMaximumAllocatableIPv4 returns the maximum amount of IPv4 addresses
 // that can be allocated to the instance
-func (n *Node) GetMaximumAllocatableIPv4() int {
+func (n *Node) GetMaximumAllocatableIPv4(scopedLog *logrus.Entry) int {
 	if n == nil {
-		log.Warningf("Could not determine first interface index, %s", getMaximumAllocatableIPv4FailureWarningStr)
+		log.Warningf("%s: %s", errUnableToDetermineFirstInterfaceIndex, getMaximumAllocatableIPv4FailureWarningStr)
```

Comment:
This seems unnecessary to me. I think we should keep the log msgs how they were, as log msgs and error msgs should not be used interchangeably.

Reply:
sounds good I will revert these then 👍

Reply:
done 👍